benchmarks/src/main/kotlin/chat/UserWithFriends.kt

```diff
@@ -0,0 +1,20 @@
+package chat
+
+import kotlinx.coroutines.channels.Channel
+
+class UserWithFriends(id: Long,
+                      activity : Double,
+                      messageChannel: Channel<Message>,
+                      configuration: BenchmarkConfiguration)
+    : User(id, activity, messageChannel, configuration) {
+    private lateinit var friends : List<Channel<Message>>
```

Comment:
constructor parameter?

Reply:
It can't be a constuctor parameter because when a user is created not all the other users are created too.